File: 02- February/20- Search Insert Position/20- Search Insert Position (Noura Algohary).cpp
// Author: Noura Algohary
// lower_bound returns the first occurrence of a num in nums vector or the position where it shour be if not exist 
class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        auto low = lower_bound(nums.begin(), nums.end(), target);
        return low - nums.begin();
    }
};

File: 02- February/20- Search Insert Position/20- Search Insert Position (Noura Algohary).py
# Author: Noura Algohary
# bisect_left returns the first occurrence of a num in nums vector or the position where it shour be if not exist 
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        low = bisect_left(nums, target)
        return low